olive/workflows/run/run.py

```diff
@@ -144,6 +145,10 @@ def run(config: Union[str, Path, dict], setup: bool = False, data_root: str = No
     set_default_logger_severity(config.engine.log_severity_level)
     ort.set_default_logger_severity(config.engine.ort_log_severity_level)
 
+    # set tmp dir root
+    if config.engine.tmp_dir_root is not None:
+        set_tmp_dir_root(config.engine.tmp_dir_root)
```

Comment:
it seems we can only change the tempfile.tempdir like https://docs.python.org/3/library/tempfile.html#tempfile.tempdir? so there is no need tmp_dir.py?